lambda/src/main/scala/pricemigrationengine/migrations/Newspaper2024Migration/Newspaper2024MigrationAmendment.scala

```diff
@@ -0,0 +1,89 @@
+package pricemigrationengine.migrations
```

Comment:
```suggestion
package pricemigrationengine.migrations.newspaper2024migration
```

Reply:
That was a great suggestion Kelvin, thanks for that. The individual components can now be more naturally named. This suggests that I should probably do the same for the other migrations, which are sharing the `migrations` namespace. Will do in a separate PR.